code/modules/mob/living/carbon/damage_procs.dm

```diff
@@ -7,6 +7,7 @@
 	spread_damage = FALSE,
 	wound_bonus = 0,
 	bare_wound_bonus = 0,
+	wound_clothing = TRUE,
```

Comment:
Add it to the end, reduces chance we mess with existing code

Reply:
Actually it should definitely be at the end since you didn't add it to the base level, which means this will break stuff